sympy/solvers/inequalities.py

```diff
@@ -396,7 +396,17 @@ def solve_univariate_inequality(expr, gen, relational=True):
 
     """
 
-    from sympy.solvers.solvers import solve, denoms
+    from sympy.solvers.solvers import denoms
+    from sympy.solvers.solveset import solveset
+
+    def solve_replacement(f_x, x):
```

Comment:
> The name `solve_replacement` makes sense in the context of this PR, but will it make sense a year from now in this file when people come across it?

Probably it won't, I like @smichr 's suggestion to keep the name of this function as `solve` so that we won't have to change the actual code. This function which emulates the output api of `solve` using `solveset` is of more general use than just this file hence I'm moving it to `solveset`. We will have two solves:
```python
sympy.solvers.solvers.solve # The actual old solve
sympy.solvers.solveset.solve # emulation of old solve by solveset
```